youtrack_sdk/client.py

```diff
@@ -164,6 +164,22 @@ def create_issue(self, *, issue: Issue) -> Issue:
             ),
         )
 
+    def update_issue(self, *, issue_id: str, issue: Issue, mute_update_notifications: bool = False) -> Issue:
```

Comment:
```suggestion
    def update_issue(self, *, issue_id: str, issue: Issue, mute_update_notifications: bool) -> Issue:
```
Let's make the user specify the value explicitly so as not to miss an important parameter.

Reply:
IMO, it is not important
Its an optional query parameter

Reply:
Ok, I agree.